### ISSUE TITLE
TimeLocale

### ISSUE THREAD

**Issue by skywalkerdark** [Agranom/ngx-material-timepicker#259]
ERROR NullInjectorError: R3InjectorError(AppModule)[InjectionToken TimeLocale -> InjectionToken TimeLocale -> InjectionToken TimeLocale]: 
  NullInjectorError: No provider for InjectionToken TimeLocale!

What is TimeLocale and where can I find it?

**Comment by AlexanderKiriluyk** [comment]
You can try import picker-module in your root application module by calling `setLocale()`, that should work

**Comment by Coooi** [comment]
Thank you @AlexanderKiriluyk , worked perfectly!

**Comment by OLDIN** [comment]
I got the same error in angular v9 with ivy and the setLocal method doesn't help me.
**installed version : 5.4.2**

```
ERROR NullInjectorError: R3InjectorError(AppModule)[InjectionToken TimeLocale -> InjectionToken TimeLocale -> InjectionToken TimeLocale]: 
  NullInjectorError: No provider for InjectionToken TimeLocale!
```

**Comment by OLDIN** [comment]
@Agranom Is there any quick workaround?

**Comment by Agranom** [comment]
Not sure. I'll update to 9 angular this week and check

**Comment by Agranom** [comment]
Updated the repo to ng9 and checked. Working without errors. You can see it on the demo page

**Comment by OLDIN** [comment]
@Agranom I see a commit with update. Have you update the NPM version?

**Comment by Agranom** [comment]
@OLDIN Yes. Try the latest version

**Comment by mahdiprog** [comment]
@Agranom I have the same issue on Angular@9.1 and ngx-material-timepicker@5.5.0.
but I used ngx-timepicker-field and  @AlexanderKiriluyk's solution not worked for me.

**Comment by achapman009** [comment]
> @Agranom I have the same issue on Angular@9.1 and ngx-material-timepicker@5.5.0.
> but I used ngx-timepicker-field and @AlexanderKiriluyk's solution not worked for me.
> and the v4.0.2 is working

This is happening to me as well on angular v9.1 and ngx-material-timepicker@5.5.1.

**Comment by OLDIN** [comment]
_**Temporary fix, but very badly**_

This can be fixed by explicitly specifying TIME_LOCALE in your app.module.ts:

```
{
    provide: TIME_LOCALE,
    useValue: ''
}
```

Full example with import:
```
import { ɵe as TIME_LOCALE } from 'ngx-material-timepicker'; // nice import :D

@NgModule({
  imports: [],
  providers: [
    {
      provide: TIME_LOCALE,
      useValue: ''
    }
  ],
  bootstrap: [AppComponent],
})
export class AppModule {}
```

**Comment by Agranom** [comment]
@mahdiprog can you provide with a code snippet how you use it, because it works well with ng9 for me

**Comment by Agranom** [comment]
For now it fixes by importing `NgxMaterialTimepickerModule` into a root module

**Comment by Agranom** [comment]
Seems to be fixed in v5.5.2
